qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: Send CV
+
+  Scenario: positive sending CV
+  Given form "Send CV" with blank fields
+  When I enter valid data in the field "Name"
+  And I enter valid data in the field "Email"
+  And I attach CV
+  And I ticked "I agree to the personal data processing"
+  And I click button "Send"
+  Then my CV has been send
+
+
+  Scenario: exceeding the limit of characters in the field "Name"
+  Given form "Send CV" with blank fields
+  When I enter 71 characters with space delimiter in the field "Name"
+  Then a message appears "The maximum number of characters is 70"
+
+  Scenario: entering the number format data in the field "Name"
+  Given form "Send CV" with blank fields
+  When I enter the number format data in the field "Name"
+  Then a message appears "Invalid name format"
+
+
+  Scenario: entering invalid data in the field "Email"
+  Given form "Send CV" with blank fields
+  When I enter email-adress without "@"-character in the field "Email"
+  Then a message appears "This field is required"
+
+
+  Scenario: upload a file larger than 10 Mb
```

Comment:
Perhaps you can add more test cases, for example to check format of attached CV file (positive and negative options).

Reply:
Oh, no, less is more :)